dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -64,14 +64,25 @@ inline void ParseSampleDesc(std::vector<SampleDesc>& samples_container,
   // Getting the components data
   std::string components_metadata;
   std::getline(index_file, components_metadata);
-  std::stringstream extensions_stream(components_metadata);
+  std::stringstream components_stream(components_metadata);
 
   // Reading consecutive components
   ComponentDesc component;
-  while (extensions_stream >> component.ext) {
-    DALI_ENFORCE(extensions_stream >> component.offset >> component.size,
-                 IndexFileErrMsg(index_path, line,
-                                 "size or offset corresponding to the extension not found"));
+  while (components_stream >> component.ext) {
+    if (kCurrentIndexVersion >= "v1.2") {
```

Comment:
In line 111 we check that the version is one of the ones we support. So here I think we shouldn't pretend we support 1.2 and higher, and check that it matches "v1.2" explicitly.

Reply:
Done